mypy/report.py

```diff
@@ -41,7 +41,7 @@
     (TypeOfAny.from_error, "Error"),
     (TypeOfAny.special_form, "Special Form"),
     (TypeOfAny.implementation_artifact, "Implementation Artifact"),
-])  # type: Final[collections.OrderedDict[int, str]]
```

Comment:
Again here the redundant type is kind of useful as documentation, so maybe leave it here.